api/src/certification/configuration/application/sco-blocked-access-dates-route.js

```diff
@@ -0,0 +1,67 @@
+import JoiDate from '@joi/date';
+import BaseJoi from 'joi';
+
+const Joi = BaseJoi.extend(JoiDate);
+
+import { securityPreHandlers } from '../../../shared/application/security-pre-handlers.js';
+import { scoBlockedAccessDatesController } from './sco-blocked-access-dates-controller.js';
+
+const register = async function (server) {
+  server.route([
+    {
+      method: 'PATCH',
+      path: '/api/admin/sco-blocked-access-dates/{key}',
+      config: {
+        pre: [
+          {
+            method: (request, h) =>
+              securityPreHandlers.hasAtLeastOneAccessOf([securityPreHandlers.checkAdminMemberHasRoleSuperAdmin])(
+                request,
+                h,
+              ),
+            assign: 'hasAuthorizationToAccessAdminScope',
+          },
+        ],
+        validate: {
+          params: Joi.object({
+            key: Joi.string().required(),
+          }),
+          payload: Joi.object({
+            data: Joi.object({
+              attributes: Joi.object({
+                value: Joi.date().format('YYYY-MM-DD').required(),
```

Comment:
Pourquoi enlever la timezone de la valeur ? Si une personne configure cela quelque part en Amerique du Sud je pense que il va penser à la date dans sa timezone

Autre point : je propose de s'habituer a mieux documenter nos parametres Joi en y ajoutant des `example` (un peu comme le `comment` sur nos colonnes de BDD)

```suggestion
                value: Joi.date().required().example('2020-12-31T11:00:38.133Z'),
```

Reply:
Il faut que je pousse quelques tests fonctionnel sur les timezones avant de pouvoir me pencher sur cette question

Reply:
J'ai finalement géré les timezone dans la dernière PR  #14213